lib/modules/manager/nix/artifacts.spec.ts

```diff
@@ -66,7 +63,7 @@ describe('modules/manager/nix/artifacts', () => {
   it('returns if no flake.lock found', async () => {
     const execSnapshots = mockExecAll();
     const res = await updateArtifacts({
-      packageFileName: 'flake.nix',
+      packageFileName: 'flake.lock',
```

Comment:
wrong, you pass the `flake.nix` and corresponding content instead

Reply:
should be resolved